dbt_checkpoint/dbt_docs_generate.py

```diff
@@ -6,34 +6,54 @@
 from dbt_checkpoint.utils import (
     add_config_args,
     add_dbt_cmd_args,
+    add_dbt_cmd_model_args,
+    add_filenames_args,
     extend_dbt_project_dir_flag,
     get_config_file,
     get_flags,
+    paths_to_dbt_models,
     run_dbt_cmd,
 )
 
 
-def docs_generate_cmd(
+def prepare_cmd(
+    paths: Sequence[str],
     global_flags: Optional[Sequence[str]] = None,
     cmd_flags: Optional[Sequence[str]] = None,
-    config: Dict[str, Any] = {}
+    prefix: str = "",
+    postfix: str = "",
+    models: Optional[Sequence[str]] = None,
+    config: Dict[str, Any] = {},
 ) -> List[str]:
     global_flags = get_flags(global_flags)
     cmd_flags = get_flags(cmd_flags)
+    if models:
+        dbt_models = models
+    else:
+        dbt_models = paths_to_dbt_models(paths, prefix, postfix)
```

Comment:
I tried to not reinvent the wheel here, this is heavily inspired by `dbt_run`current hook.